NetflixClone/Classes/Scenes/Home/HomeViewController.swift

```diff
@@ -6,125 +6,192 @@
 //
 
 import UIKit
+import Combine
 
-class HomeViewController: UIViewController, UICollectionViewDataSource {
-    
-    enum Defaults {
-        static let moviesPerPage = 20
-    }
+class HomeViewController: UIViewController {
+    // MARK: Properties
+    typealias Section = HomeViewModel.Section
     
-    // MARK: Outlets
+    private var viewModel: HomeViewModel
+    private var subscribers: [AnyCancellable] = []
```

Comment:
```suggestion
    private var subscribers: Set<AnyCancellable> = []
```
It should be a Set, not a List; Since it helps random item selection.